### PR TITLE
Fix an inversion between username and password

### PR DESCRIPTION
**Pull request by gthvn1**

### CODE REVIEW

File: metadata/service/server/init.yml [hunk 1]
```diff
@@ -4,8 +4,8 @@ classes:
 - service.nagios.support
 parameters:
   _param:
-    nagios_password: r00tme
     nagios_username: nagiosadmin
+    nagios_password: r00tme
```
Comment (fpytloun) [inline metadata/service/server/init.yml]: You should not define any default password in service metadata to force user define it (or fail on UndefinedVariable error) on it's own on system level.

Comment (gthvn1) [reply metadata/service/server/init.yml]: I agree. So a better fix should be to just remove **nagios_username** and **nagios_password** right?

Comment (fpytloun) [reply metadata/service/server/init.yml]: Simply remove definition of `_param:nagios_password` from here.

Comment (gthvn1) [reply metadata/service/server/init.yml]: And more generally it is not a good practice to define parameters in *_param* into service metadata right? I mean parameters under *_param* should be defined only at the system level?

**Comment by fpytloun** [comment]
Yes, absolutely, you should not define _param on service level at all, this should be only system/cluster-level specific feature (service level should be straightforward and as simple as possible).

**Pull request merged**